contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -401,116 +348,6 @@ contract VaultAdmin is VaultStorage {
         IERC20(_asset).safeTransfer(governor(), _amount);
     }
 
-    /**
```

Comment:
this code getting deleted from the vault makes me so happpppppyyyyyyy!

Reply:
🎉 🎉 🎉